__tests__/index.test.js

```diff
@@ -93,3 +93,34 @@ describe("GET /api/articles/:article_id", () => {
 
 });
 
+describe("GET /api/articles", () => {
+  test("status 200: responds with an array of article objects", () => {
+      return request(app)
+      .get("/api/articles")
+      .expect(200)
+      .then(({ body }) => {
+          expect(body.articles.length).toBe(13);
+          body.articles.forEach((article) => {
+              expect(typeof article.author).toBe("string");
+              expect(typeof article.title).toBe("string");
+              expect(typeof article.article_id).toBe("number");
+              expect(typeof article.topic).toBe("string");
+              expect(typeof article.created_at).toBe("string");
+              expect(typeof article.votes).toBe("number");
+              expect(typeof article.article_img_url).toBe("string");
+              expect(typeof article.comment_count).toBe("string");
+          });
+      });
+  });
+
+  //Error handling
+  test("status 404: responds with error message when given a wrong URL", () => {
+      return request(app)
```

Comment:
We only need one test for the 404 as it's for all our endpoints. You already this before so this test can be removed.